app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewScreen.kt

```diff
@@ -0,0 +1,146 @@
+package com.kickstarter.features.pledgedprojectsoverview.ui
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListState
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.kickstarter.R
+import com.kickstarter.ui.compose.designsystem.KSTheme
+import com.kickstarter.ui.compose.designsystem.KSTheme.colors
+import com.kickstarter.ui.compose.designsystem.KSTheme.dimensions
+import com.kickstarter.ui.compose.designsystem.KSTheme.typography
+import com.kickstarter.ui.toolbars.compose.TopToolBar
+
+@Composable
+@Preview(name = "Light", uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(name = "Dark", uiMode = Configuration.UI_MODE_NIGHT_YES)
+private fun PledgedProjectsOverviewScreenPreview() {
+    KSTheme {
+        Scaffold(
+            backgroundColor = colors.backgroundAccentGraySubtle
+        ) { padding ->
+            PledgedProjectsOverviewScreen(
+                modifier = Modifier.padding(padding),
+                lazyColumnListState = rememberLazyListState(),
+                ppoCards = (0..10).map {
+                    PPOCardDataMock()
+                },
+                alertsAmount = 10,
+                onBackPressed = {}
+            )
+        }
+    }
+}
+
+@Composable
+fun PledgedProjectsOverviewScreen(
+    modifier: Modifier,
+    alertsAmount: Int = 0,
+    onBackPressed: () -> Unit,
+    lazyColumnListState: LazyListState,
+    ppoCards: List<PPOCardDataMock> = listOf(),
+) {
+    Box(
+        modifier = Modifier.fillMaxSize(),
+        contentAlignment = Alignment.Center
+    ) {
+        Scaffold(
+            modifier = modifier,
+            topBar = {
+                TopToolBar(
+                    title = stringResource(id = R.string.project_alerts_fpo),
+                    titleColor = colors.kds_support_700,
```

Comment:
This is an old color that we might want to try to change to one of the new colors? https://github.com/kickstarter/android-oss/blob/a0eb1a32685f1678b437d617b559a29aaf23eb05/app/src/main/java/com/kickstarter/ui/compose/designsystem/KSColors.kt#L42

Reply:
good catch, thanks!